cmd/pitr/pxc/pxc.go

```diff
@@ -75,32 +75,40 @@ func (p *PXC) GetGTIDSet(binlogName string) (string, error) {
 	return binlogSet, nil
 }
 
+type Binlog struct {
+	Name      string
+	Size      int64
+	Encrypted string
+}
+
 // GetBinLogList return binary log files list
-func (p *PXC) GetBinLogList() ([]string, error) {
+func (p *PXC) GetBinLogList() ([]string, []Binlog, error) {
 	var binlogList []string
 	rows, err := p.db.Query("SHOW BINARY LOGS")
 	if err != nil {
```

Comment:
it it's possible that this query will return `sql.ErrNoRows` error?
I think in that case we should not return error

Reply:
no need to continue if we don't have binlogs list

Reply:
yea, but in that case we will return an error
I mean can we return all nils and do nothing instead of writing error that we can't find anything

Reply:
if binlogs-collector can't find any binlogs, I think this is error, and user should know about this

Reply:
kk then)